kalite/distributed/management/commands/setup.py

```diff
@@ -263,7 +264,20 @@ def handle(self, *args, **options):
         ########################
         # Now do stuff
         ########################
-
+        
+        # Clean *pyc files if we are in a git repo
+        if not install_clean and settings.IS_SOURCE:
+            if os.access(settings.SOURCE_DIR, os.W_OK):
+                for root, dirs, files in os.walk(settings.SOURCE_DIR):
+                    pyc_files = filter(lambda filename: filename.endswith(".pyc"), files)
+                    py_files = set(filter(lambda filename: filename.endswith(".py"), files))
+                    excess_pyc_files = filter(lambda pyc_filename: pyc_filename[:-1] not in py_files, pyc_files)
+                    for excess_pyc_file in excess_pyc_files:
+                        full_path = os.path.join(root, excess_pyc_file)
+                        os.remove(full_path)
```

Comment:
(I still got the "CommandError: Unknown command: 'migrate'" error while running setup)